tests/data/ec2_offer.py

```diff
@@ -1,424 +1,534 @@
+            "publicationDate": "2019-03-29T04:06:54Z"
+        },
+        {
+            "product": {
+                "productFamily": "Compute Instance",
+                "attributes": {
+                    "enhancedNetworkingSupported": "Yes",
+                    "memory": "61 GiB",
+                    "dedicatedEbsThroughput": "1750 Mbps",
+                    "vcpu": "8",
+                    "capacitystatus": "AllocatedCapacityReservation",
+                    "locationType": "AWS Region",
+                    "storage": "EBS only",
+                    "instanceFamily": "GPU instance",
+                    "operatingSystem": "Linux",
+                    "physicalProcessor": "Intel Xeon E5-2686 v4 (Broadwell)",
+                    "clockSpeed": "2.3 GHz",
+                    "ecu": "26",
+                    "networkPerformance": "Up to 10 Gigabit",
+                    "servicename": "Amazon Elastic Compute Cloud",
+                    "instancesku": "AF82PTVU4BZDKAPW",
+                    "instanceType": "p3.2xlarge",
+                    "tenancy": "Dedicated",
+                    "usagetype": "CAN1-DedicatedRes:p3.2xlarge",
+                    "normalizationSizeFactor": "16",
+                    "gpu": "1",
+                    "processorFeatures": "Intel AVX, Intel AVX2, Intel Turbo",
+                    "servicecode": "AmazonEC2",
+                    "licenseModel": "No License required",
+                    "currentGeneration": "Yes",
+                    "preInstalledSw": "NA",
+                    "location": "Canada (Central)",
+                    "processorArchitecture": "64-bit",
+                    "operation": "RunInstances"
+                },
+                "sku": "23HJ4NUSYQBPWZQN"
+            },
+            "serviceCode": "AmazonEC2",
+            "terms": {
+                "OnDemand": {
+                    "23HJ4NUSYQBPWZQN.JRTCKXETXF": {
+                        "priceDimensions": {
+                            "23HJ4NUSYQBPWZQN.JRTCKXETXF.6YS6EN2CT7": {
+                                "unit": "Hrs",
+                                "endRange": "Inf",
+                                "description": "$0.00 per Dedicated Reservation Linux p3.2xlarge Instance Hour",
+                                "appliesTo": [],
+                                "rateCode": "23HJ4NUSYQBPWZQN.JRTCKXETXF.6YS6EN2CT7",
+                                "beginRange": "0",
+                                "pricePerUnit": {
+                                    "USD": "0.0000000000"
+                                }
+                            }
+                        },
+                        "sku": "23HJ4NUSYQBPWZQN",
+                        "effectiveDate": "2019-03-01T00:00:00Z",
+                        "offerTermCode": "JRTCKXETXF",
+                        "termAttributes": {}
+                    }
+                }
+            },
+            "version": "20190329040654",
```

Comment:
Example of per product pricing version (and overall example of a page of query API results)